.github/workflows/code-scanner.yaml

```
@@ -21,6 +21,11 @@ on:
       - develop
       - main
 
+  pull_request:
```

actionlint

key "pull_request" is duplicated in "on" section. previously defined at line:9,col:3

(syntax-check)